### PR TITLE
Add org.telesculptor.TeleSculptor

### PR DESCRIPTION
**Pull request by kevinsmia1939**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. https://github.com/Kitware/TeleSculptor/issues/425
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by kevinsmia1939** [comment]
bot, build org.telesculptor.TeleSculptor

File: org.telesculptor.TeleSculptor.desktop [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```
Comment (barthalion) [inline org.telesculptor.TeleSculptor.desktop:1]: Should be ideally upstreamed.

File: org.telesculptor.TeleSculptor.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline org.telesculptor.TeleSculptor.metainfo.xml:1]: Should be ideally upstreamed.

**Comment by barthalion** [comment]
From terminal:

```
WARN - plugin_loader::Unable to load shared library "/app/bin/../lib/kwiver/plugins/algorithms/kwiver_algo_ceres_plugin.so" : liblapack.so.3: cannot open shared object file: No such file or directory
```

File: org.telesculptor.TeleSculptor.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+app-id: org.telesculptor.TeleSculptor
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: TeleSculptor
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
```
Comment (barthalion) [inline org.telesculptor.TeleSculptor.yaml:9]: It doesn't seem to use portals so it need at least `--filesystem=home`.

File: org.telesculptor.TeleSculptor.desktop (hunk 1 above)
Comment (kevinsmia1939) [reply org.telesculptor.TeleSculptor.desktop:1]: I am working with them on this.

File: org.telesculptor.TeleSculptor.metainfo.xml (hunk 2 above)
Comment (kevinsmia1939) [reply org.telesculptor.TeleSculptor.metainfo.xml:1]: I am working with them on this.

**Comment by kevinsmia1939** [comment]
bot, build org.telesculptor.TeleSculptor

**Comment by kevinsmia1939** [comment]
bot, build org.telesculptor.TeleSculptor

**Comment by kevinsmia1939** [comment]
Patch proposal for desktop and metainfo.
https://github.com/Kitware/TeleSculptor/pull/536

**Comment by kevinsmia1939** [comment]
bot, build org.telesculptor.TeleSculptor

File: org.telesculptor.TeleSculptor.yaml (hunk 3 above)
Comment (kevinsmia1939) [reply org.telesculptor.TeleSculptor.yaml:9]: I add it.

Comment (kevinsmia1939) [reply org.telesculptor.TeleSculptor.yaml:9]: and add lapack

**Comment by kevinsmia1939** [comment]
it should be good now

**Pull request closed without merging**